Assets/BossRoom/Scripts/Server/Game/Entity/ServerProjectileLogic.cs

```diff
@@ -110,15 +110,15 @@ private void DetectCollisions()
             for (int i = 0; i < numCollisions; i++)
             {
                 int layerTest = 1 << m_CollisionCache[i].gameObject.layer;
-
-                if ((layerTest & m_BlockerMask) != 0)
-                {
-                    //hit a wall; leave it for a couple of seconds.
-                    m_ProjectileInfo.Speed_m_s = 0;
-                    m_IsDead = true;
-                    m_DestroyAtSec = Time.fixedTime + k_WallLingerSec;
-                    return;
-                }
+                //Debug.Log(m_CollisionCache.Length);
+                //if ((layerTest & m_BlockerMask) != 0)
+                //{
+                //    //hit a wall; leave it for a couple of seconds.
+                //    //m_ProjectileInfo.Speed_m_s = 0;
+                //    m_IsDead = true;
+                //    m_DestroyAtSec = Time.fixedTime;
+                //    return;
+                //}
```

Comment:
From Ryan: For the pooling of arrows, I removed it from the mage bolt and figured out why the arrows stopped moving; the collision check keeps setting the velocity to 0, it seems to not be using the correct mask values? I disabled it and now they're back to working as before, and polling, but no longer collide with walls.
Sam: we'll need to figure this out and reenable collision checks